src/PlatformSockets.c

```diff
@@ -10,24 +6,18 @@
 
 void addrToUrlSafeString(struct sockaddr_storage* addr, char* string)
 {
-    char addrstr[48];
-#ifdef __vita__
-    struct sockaddr_in* sin = (struct sockaddr_in*)addr;
-    sceNetInetNtop(addr->ss_family, &sin->sin_addr, addrstr, sizeof(addrstr));
-    if (addr->ss_family == AF_INET6) {
-        sprintf(string, "[%s]", addrstr);
-    } else {
-        sprintf(string, "%s", addrstr);
-    }
-#else
     char addrstr[INET6_ADDRSTRLEN];
 
     if (addr->ss_family == AF_INET6) {
+#ifdef __vita__
+        LC_ASSERT(FALSE);
```

Comment:
right.
i just split `sockaddr_in` & `sockaddr_in6` cases.
least, we need this.
cuz sockadder_in6 not defined both vita-header & vita-newlib :)